polylogue/index.py

```
@@ -38,7 +38,10 @@ def __init__(self) -> None:
         try:
             from .index_qdrant import update_qdrant_index
         except ImportError as exc:  # pragma: no cover
-            raise RuntimeError("POLYLOGUE_INDEX_BACKEND=qdrant requires the 'qdrant-client' package.") from exc
+            raise RuntimeError(
+                "POLYLOGUE_INDEX_BACKEND=qdrant requires the 'qdrant-client' package. "
+                "Install it with: pip install qdrant-client"
+            ) from exc
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)